swap/types.go

```diff
@@ -49,3 +54,120 @@ type EmitChequeMsg struct {
 type ConfirmChequeMsg struct {
 	Cheque *Cheque
 }
+
+// Uint256 represents an unsigned integer of 256 bits
```

Comment:
It looks to me that Uint256 implementation could be in its own file, or even package as it is already exposed.

Reply:
thanks for the suggestion, this should be implemented through #2076